app/src/components/AddCredentialButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useCallback} from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, TouchableOpacity } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
+import { useTheme, testIdWithKey, useStore } from 'aries-bifold'
+import { BCState, BCDispatchAction } from '../store'
+
+const AddCredentialButton: React.FC = () => {
+    const { t } = useTranslation()
+    const { ColorPallet } = useTheme()
+    const [, dispatch] = useStore<BCState>()
+
+    const activateSlider = useCallback(() => {
+        dispatch({
```

Comment:
Does this button component need to communicate over multiple other components? It would be better to just pass in an `onPress` fn if possible. Usually `dispatch` is best used when its overly complicated to pass in callbacks.

Reply:
The AddCredentialButton is located in the navigation header while the component that it effects is located in the credential list screen. I could pass in an on press, but I'd have to do that in the index.ts file and I don't feel like that conventional